lib/fortnox/api/models/customer.rb

```diff
@@ -161,7 +161,7 @@ class Customer < Fortnox::API::Model::Base
         attribute :project, Types::Nullable::String
 
         # SalesAccount	Sales account of the customer, 4 digits
-        attribute :sales_account, Types::AccountNumber
+        attribute :sales_account, Types::Sized::String[4]
```

Comment:
This is not ok as is as it would allow the account NUMBER to be set to `"abcd"` etc. If the API actually barfs if we send it a number here we should do some casting instead.
Is this something you have experienced problems with or was it an attempt to prevent problems? Also note that we do NOT reflect the Fortnox API spec directly in our API due to theirs being full of errors, conflicts and inconsistencies. Instead we try to normalise the API as much as possible, returning actual Ruby Date objects for date and time, always representing numeric types as numbers, using enums in place of strings for countries etc.